### ISSUE TITLE
PublicKeyCredential - Unrecognized Field

### ISSUE THREAD

**Issue by adamberryhuff** [Yubico/java-webauthn-server#145]
Hello Emil and team,

One other thing I found in the JavaScript readme samples is that we include `transports` in the `publicKeyCredential` object [here](https://github.com/Yubico/java-webauthn-server/blob/master/README#L241). 

I'm not sure if this is a bug or if I just have a different version of the Java package but, for me, this results in the following error:
```
UnrecognizedPropertyException: Unrecognized field \"transports\" (class com.yubico.webauthn.data.AuthenticatorAttestationResponse), not marked as ignorable
```

Further investigation into the documentation indicates that this object does not accept this param:
https://developers.yubico.com/java-webauthn-server/JavaDoc/webauthn-server-core/latest/com/yubico/webauthn/data/PublicKeyCredential.html

I have a feeling this may have been added in the pre-release but just wanted to follow up just in case that's not the case.

Thanks!

**Comment by emlun** [comment]
Hi! Yes, that's being added in version `1.11.0`.

For the record: see also issue #141

**Comment by emlun** [comment]
The `transports` field is now released in version `1.11.0`, and the release procedure has been tweaked to avoid this kind of confusion in the future. Thanks again!
